Update Integration asset table App_uuid (#20830)
DataDog/documentation: layouts/shortcodes/integration-assets-reference.md

| Attribute                                            | Mandatory or Optional       | Description                                                                                                                                                                                                                                                                       |
|----------|----------|----------|
| `manifest_version`                                   | Mandatory                   | **Type**: String Enum.<br><br>Version of the manifest schema. Supported values include `1.0.0` and `2.0.0`.                                                                                                                                                                       |

> | `app_uuid`                                           | Mandatory                   | **Type**: UUID.<br><br>Globally unique UUID for this application. This is automatically generated by Datadog when running the `ddev create` command during the process of creating your tile scaffolding for your integration tile.                                                                                                                                                                                                                   |

Author: Can we link https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#populate-the-integration-tile-scaffolding  to ddev create?

> | `app_uuid`                                           | Mandatory                   | **Type**: UUID.<br><br>Globally unique UUID for this application. This is automatically generated by Datadog when running the developer toolkit `ddev create` command during the process of creating your tile scaffolding for your integration tile.                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| `app_uuid`                                           | Mandatory                   | **Type**: UUID.<br><br>Globally unique UUID for this application. This is automatically generated by Datadog when the developer toolkit `ddev create` command runs as part of creating tile scaffolding for your integration tile.                                                                                                                                                                                                                   |
```
